WordPress/src/main/java/org/wordpress/android/models/Blog.java

```diff
@@ -497,7 +499,30 @@ public void setPlanID(long planID) {
     public String getPlanShortName() {
         return StringUtils.notNullStr(planShortName);
     }
+
     public void setPlanShortName(String name) {
         this.planShortName = StringUtils.notNullStr(name);
     }
+
+    public String getCapabilities() {
+        return capabilities;
+    }
+
+    public void setCapabilities(String capabilities) {
+        this.capabilities = capabilities;
+    }
+
+    public boolean checkCapability(Capability capability) {
+        // If a capability is missing it means the user don't have it.
+        if (capabilities == null || capabilities.isEmpty() || capability == null) {
+            return false;
+        }
+        try {
+            JSONObject jsonObject = new JSONObject(capabilities);
```

Comment:
I haven't considered before, but thinking about it now, I don't think I agree that we're going to check the "capabilities" as often as you think. Aside from People Management, I think we're only going to check `list_users` for once. Also, we tend to read the blog from db a lot and if we stored it as `JSONObject` we'd do that expensive operation even when it's not used.

I think a middle ground solution here would be to just cache the capabilities as a `JSONObject` when it's first requested, so we avoid the situations where we query the db for a network request or something (rather than building the UI). What do you think?